packages/java-parser/test/samples-spec.js

```diff
@@ -10,7 +10,7 @@ const javaParser = require("../src/index");
 describe("The Java Parser", () => {
   createSampleSpecs("java-design-patterns");
   createSampleSpecs("spring-boot");
-  createFailingSampleSpec("guava", 22, 0);
+  createFailingSampleSpec("guava", 10, 0);
```

Comment:
good, almost there for the Guava code 👍 

All our three samples together are now close to 700,000 Lines of code (excluding comments and blank lines).
So I think the next steps would be to start actually implementing he prettier printer.

There is a full E2E example in Toml Prettier Plugin
- https://github.com/bd82/toml-tools/tree/master/packages/prettier-plugin-toml

And place-holders in this repo's prettier-plugin
With a micro debug able example:
https://github.com/jhipster/prettier-java/tree/master/packages/prettier-plugin-java/test/hello-world